src/js/zqd/zqd.js

```diff
@@ -81,18 +81,14 @@ export class ZQD {
   start() {
     mkdirSync(this.root, {recursive: true, mode: 0o755})
 
-    // PATH must include both the zqd and zeek bin directories, as zqd depends
-    // on having zeek available in its environment PATH.
+    // We saw errors on command.com vs powershell when we tried to clone
+    // process.env and then determine whether to use "PATH" or "Path".
+    // Directly altering process.env is safe and less error prone.
```

Comment:
I didn't understand why, but the note at https://nodejs.org/dist/latest-v12.x/docs/api/process.html#process_process_env helps. Mind linking to that or repeating the claim that Windows environment variables are case-insensitive?